README.md

```diff
@@ -1,3 +1,46 @@
-# Jellyfin Chromecast
+<h1 align="center">Jellyfin Chromecast Web Receiver</h1>
```

Comment:
```suggestion
<h1 align="center">Jellyfin Cast Web Receiver</h1>
```

Chromecast is renamed to Google Cast or just Cast these days, so let's use that for the name (even though the repository url still uses the chromecast name)